README.md

```diff
@@ -1,158 +1,16 @@
-# chart-verifier
+# `chart-verifier`: Rules based tool to certify Helm Charts
 
-`chart-verifier` is a tool that validates a Helm chart against a configurable list of checks; individual checks can be
-included or excluded through command line options. The default set of tests covers Red Hat’s recommendations.
+## Overview 
 
-Each check is independent and execution order is not guaranteed. Input is provided through options in
-the command line interface; currently the only input is the required `uri` option.
+The **chart-verifier** CLI tool allows you to validate the Helm chart against a configurable list of checks. The tool ensures that the Helm Charts include the associated metadata and formatting, and are distribution ready.
 
-The following checks have been implemented:
+The tool allows users to validate a Helm Chart URL and provides a report where each check has a `positive` or `negative` result. A negative result from a check indicates a problem with the chart which needs correction. It ensures that the Helm chart works seamlessly on Red Hat OpenShift and can be submitted as a certified Helm Chart in the [Red Hat Helm Repository](https://github.com/openshift-helm-charts).
 
-| Name | Description
-|---|---
-| `is-helm-v3` | Checks whether the given `uri` is a Helm v3 chart.
-| `has-readme` | Checks whether the Helm chart contains a `README.md` file.
-| `contains-test` | Checks whether the Helm chart contains at least one test file.
-| `has-minkubeversion` | Checks whether the Helm chart's `Chart.yaml` includes the `minKubeVersion` field.
-| `readme-contains-values-schema` | Checks whether the Helm chart `README.md` file contains a `values` schema section.
-| `not-contains-crds` | Check whether the Helm chart does not include CRDs.
+The input is provided through the command-line interface, with the only required input parameter being the `uri` option. The output is represented through a YAML format with descriptions added in a human-readable format. The report should be submitted with a full set of checks thus validating the Helm Chart URL.
```

Comment:
Please confirm with Baiju that `uri` option is correct. In Sri's doc draft he mentioned that URL would be appropriate. But again this is an option so `uri` may be right.

Reply:
`uri` fine as the name of the option.